pkg/accountGRPC/controller/controller.go

```diff
@@ -0,0 +1,174 @@
+package accountgrpccontroller
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	customerrors "github.com/stasBigunenko/monorepa/errors"
+	"github.com/stasBigunenko/monorepa/model"
+	pb "github.com/stasBigunenko/monorepa/pkg/accountGRPC/proto"
+)
+
+type AccountGRPCСontroller struct {
+	client pb.AccountGRPCServiceClient
+}
+
+func New(cli pb.AccountGRPCServiceClient) AccountGRPCСontroller {
```

Comment:
You probably want to return pointer.

Reply:
Pointers are returned when we are supposed to modify the object, direct obj is returned where we the inner state in immutable. Here its the second case

Reply:
You are mutating `pb.AccountGRPCServiceClient` grpc interface with methods such as `UpdateAccount. It is making network request which triggers a lot of state changing inside grpc.